src/inspector/models/score/scoreaccessibilitysettingsmodel.cpp

```diff
@@ -0,0 +1,150 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "scoreaccessibilitysettingsmodel.h"
+
+#include "translation.h"
+#include "log.h"
+
+using namespace mu::inspector;
+using namespace mu::notation;
+
+ScoreAccessibilitySettingsModel::ScoreAccessibilitySettingsModel(QObject* parent, IElementRepositoryService* repository)
+    : AbstractInspectorModel(parent, repository)
+{
+    setSectionType(InspectorSectionType::SECTION_SCORE_ACCESSIBILITY);
+    setTitle(muse::qtrc("inspector", "Accessibility"));
+    createProperties();
+    connect(m_scoreStylePreset, &PropertyItem::valueChanged, this, [this]() {
+        if (!m_ignoreStyleChange) {
+            setScoreStylePreset(m_scoreStylePreset);
```

Comment:
The unusual use of `PropertyItem*` also leads to a very complicated code flow here. So, when the user clicks an option in the dropdown, this code is called...